views/conversation.pug

```diff
@@ -11,6 +11,30 @@ block page_name
   = _("Chat")
 
 block content
+  div.modal#comment-popup
+    div.modal-dialog
+      form(method='post')
+        div.modal-content
+          div.modal-body
+            div.form-group
+              label(for='comment-block').control-label= _("Write your comment for the last response")
+              textarea.form-control#comment-block
+          div.modal-footer
+            div.form-group
+              button(type='submit').btn.btn-primary= _("Comment")
+
+  div#toolbar
+    div.form
+      div.row
+        div.col
+          label(for='recording-toggle').control-label
+          input(type='checkbox')#recording-toggle
+          = _(" Recording conversation")
```

Comment:
Is this going to be gated by some setting in Configuration, or always available?
I want, somewhere, to explain what recording does, and explain it is not the same as the usual passive recording we do. Privacy implications are very different, and "recording mode" is not covered by IRB, so one must never use it with personal accounts or private date.

Reply:
I added a warning before confirmed (which gets me struggled with toggle the checkbox...) 
If you can send me a paragraph you want to show to the user, it would be great :)

Reply:
I'm not sure exactly what to write. Something along the lines of
"""
While recording is active, all your commands and all the replies from Almond will be stored. The recording will include your account and device names, as well as the results from performing actions on your accounts. The recording will be stored locally and you will have the option to download it. Please do not include any private information in any recording you plan to share with others (e.g. in a bug report).
"""

It's a bit long to show all the time though. How are we showing this?

Reply:
Also, don't put our emails in the recording message. We'll explain in the documentation how to produce recordings for testing, but we don't want to imply that this function is only for bug reports.